CHANGELOG.md

```diff
@@ -8,6 +8,40 @@
   replaced by the stable `io.opentelemetry.instrumentation.api.semconv.util.SpanNames`
 - In preparation for stabilizing HTTP library instrumentation, the classes and methods
   that were deprecated in the prior two releases have now been removed
+  ([#13135](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13135),
+   [#13150](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13150))
+- Deprecated Dubbo instrumentation methods were removed
+  ([#13076](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13076))
+
+### 📈 Enhancements
+
+- Add Lettuce metrics
```

Comment:
maybe rephrase to `Add db client metrics to lettuce instrumentation`

Reply:
```suggestion
- Add database client metrics to Lettuce instrumentation
```